src/mpi/coll/include/coll_impl.h

```diff
@@ -21,6 +21,8 @@
 #include "../algorithms/treealgo/treealgo.h"
 #include "../algorithms/recexchalgo/recexchalgo.h"
 
+#include "csel_container.h"
+
 #define MPII_COLLECTIVE_FALLBACK_CHECK(check)                           \
     do {                                                                \
         if ((check) == 0) {                                             \
```

Comment:
https://github.com/pmodels/mpich/blob/03c20bb9bd61b55b344e243c56a309c8eadaf1b9/src/mpi/coll/include/coll_impl.h#L32

can we change this line to only print on rank = 0? When we ran the auto tuner, there are too many prints.

Reply:
Fixed.